packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -56,11 +57,11 @@ export class FlowBuilderApi {
     return node as T
   }
 
-  getNodeByCode(code: string): HtNodeComponent {
+  getNodeByContentID(contentID: string): HtNodeComponent {
```

Comment:
why using uppercase `D` instead of `Id` as we do with other variables?

Reply:
I have used ContentID because that is how it is written in the flow builder frontend